cohort_3/curriculum/workshops/portfolio/README.md

```diff
@@ -0,0 +1,271 @@
+      username). Make sure `Initialize this repository with a README` **is not**
+      checked.
+
+      ![](img/github_create_repo.gif)
+
+    - To use GitHub, we need to use another tool called git. Git is a tool that
+      lets us take "snapshots" of the code we've written so far. GitHub is a
+      place where we can store our git projects and collaborate with others. Do
+      note, GitHub and git are _not_ the same thing. We'll go further into the
+      distinction later.
+    - Git is already installed on Cloud9. Go ahead and open a new terminal tab
+      and run the command `git init` to create a new git project.
+
+      ![](img/git_init.gif)
+
+    - Go ahead and run `git add -A :/` and then `git commit -m "Initial
+      commit."` in the terminal. Don't worry about what these two commands mean,
+      they're just telling git to create a snapshot of our code and adding the
+      message "Initial commit." to the snapshot.
+
+      ![](img/git_initial_commit.gif)
+
+    - Final step. Let's tell git to upload our code to GitHub. Run the following
+      two commands in your terminal, replacing `{username}` with your GitHub
+      username. It'll prompt you for your GitHub username and password, go ahead
+      and enter those. Don't worry if it doesn't display any characters while
+      you type your password, just hit enter when you're done and all should be
+      well with the world.
+
+            $ git remote add origin https://github.com/{username}/{username}.github.io.git
+            $ git push -u origin master
+
+      ![](img/git_initial_push.gif)
+
+    - Great! Now if you head over to
+      https://{username}.github.io/portfolio/index.html (replace {username} with
+      your GitHub username), you'll see the site we just made deployed live. It
+      may take a few minutes for GitHub to prepare your site. If you get an
+      error, no worries, just wait a few minutes and try again.
+- What have we done so far?
+    - So what was that code we typed? Spoiler alert: almost every website on the
+      internet is secretly written in a language called **HyperText Markup
+      Language**, otherwise known as "HTML". So why don't you see code
+      everywhere? Well, your browser reads HTML for you and just shows you the
+      pretty version. It is sheltering you!
+        - _Warning: this wording was taken from Codecademy - remember to change
+          this_
+    - We'll get our hands a bit dirtier with the basics of HTML. We'll talk a
+      little about formatting, links, and images so you can make a simple, but
+      ugly, webpage.
+        - _Warning: this wording was taken from Codecademy - remember to change
+          this_
+    - If you want to make it pretty, we'll also do a quick dive into Cascading
+      Syle Sheet (CSS), which many websites secretly use to make themselves
+      pretty.
+    - The basic building blocks of HTML are _HTML elements_. HTML elements are
```

Comment:
This should be right after them working on HTML. It will be forgotten by the time they put their site on gh-pages